foscmd

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+if __name__ == '__main__':
+    import argparse
+    import json
+    from foscam import FoscamCamera
+
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument("-H", "--host", help="camera host/ip")
+    parser.add_argument("-P", "--port", help="camera port", type=int)
+    parser.add_argument("-u", "--username", help="camera username")
+    parser.add_argument("-p", "--password", help="camera password")
+    parser.add_argument("-v", "--verbose", action="store_true",
+                    help="increase output verbosity")
+    args = parser.parse_args()
+
+    if args.verbose:
+        print("verbosity turned on")
+
+    print(json.dumps(FoscamCamera(args.host, args.port, args.username, args.password,verbose=args.verbose).get_dev_state()[1]))
```

Comment:
@texnofobix Thanks for you pr. But it may be not useful for other users. It's too special.
I think you can rewrite you script and make it  a common cli-tool which supports all of the foscam APIs. Just like the [python-onvif cli](https://github.com/quatanium/python-onvif#onvif-cli)